client/my-sites/migrate/components/import-type-choice/style.scss

```diff
@@ -5,14 +5,15 @@
 		justify-content: flex-start;
 		align-items: flex-start;
 
-		padding: 20px;
+		padding: 16px;
 
 		cursor: pointer;
 
 		border: 1px solid var( --color-border ); /* TODO fix color */
```

Comment:
@danhauk Any option on what the correct color to use here is?

Reply:
I think `--color-border` is a great option to keep it consistent with other UI element borders as described in [the Color section of devdocs](https://wpcalypso.wordpress.com/devdocs/docs/color.md). That way if changes are made to UI element borders this changes with it and we won't have to manually update that.